workshops/py_Christmas/README.md

```diff
@@ -0,0 +1,381 @@
+
+Using the coding space on the left create a file and name it 'Figures.py' (most probably a main.py file will already be created, don't use that).
+Now in the coding window :
+
+First let's import the turtle library :
+```python
+import turtle
+```
+
+Next, let's make a function to draw a rectangle 
+
+```python
+def box1 (turtle, color, x, y, width, height):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x,y)
+    turtle.pendown()
+    turtle.begin_fill()
+    for i in range(2):
+        turtle.forward(width)
+        turtle.left(90)
+        turtle.forward(height)
+        turtle.left(90)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+```
+   Was that a bit too much to take in ? No worries,I will  break it down for you. 
+```python
+def box1 (turtle, color, x, y, width, height):
+```
+First we create the function definition with color of the rectangle (color) , postion of the turtle/where to start drawing the rectangle from (x and y and coordinates) and finally width and height of the box/rectangle.
+
+```python
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x,y)
+    turtle.pendown()
+```
+We use ******penup()****** and ******pendown()****** function to move the turtle without drawing anything (just like lifting the pen from the paper and keeping back on paper)
+******goto(x,y)****** decides the position of the turtle
+```python
+for i in range(2):
+        turtle.forward(width)
+        turtle.left(90)
+        turtle.forward(height)
+        turtle.left(90)
+```
+The code above is the one that draws the rectangle. This code moves the turtle to on side by the same distance as the width mentioned, now turns it to left (left of the turtle) and moves it to a distance same as the height. This process is repeated twice as it is put in a loop twice. Now we have a complete rectangle. 
+Are you still with me ? 
+
+Let's make the star now shall we ? Check the code below and see if you understand
+
+```python
+def star(turtle, color, x, y, size):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    for i in range (5):
+        turtle.forward(size)
+        turtle.right(144)
+        turtle.forward(size)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+```
+I guess you would've understood atleast 90% of the above code easily. I will explain the new stuff.
+The ******size****** parameter decides the legth of the edges of the star.
+Now we move the turtle forward by a distance same as that of the size mentioned. Now we turn the turtle to right by 144 degreesand move it by the same distance. Repeate the step 5 times and you will have a star. Got it ? 
+
+Last function in this file to make the tiny balls on either sides of the boxes (green part) pf the christmas tree.
+Go through the code. You probably will understand it pretty easily.
```

Comment:
```suggestion
```
I guess you would've understood at least 90% of the above code easily. I will explain the new stuff.
The ******size****** parameter decides the length of the edges of the star.
Now we move the turtle forward by a distance the same as that of the size mentioned. Now we turn the turtle to right by 144 degrees and move it by the same distance. Repeat the step 5 times and you will have a star. Got it? 

The last function in this file to make the tiny balls on either side of the boxes (green part) of the Christmas tree.
Go through the code. You probably will understand it pretty easily.
```